change/@fluentui-react-dialog-cedc6c7c-fd4d-4c67-bdec-52e9129f7a2d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Dialog is now in released
```suggestion
  "type": "patch",
```